drivers/windows/R/bootstrap.R

```diff
@@ -1,31 +1,68 @@
-bootstrap_update <- function(development = NULL, root = NULL) {
+bootstrap_update <- function(development = NULL, root = NULL,
+                             platform = "windows") {
   path_script <- "hipercow/bootstrap-windows.R"
   path_root <- hipercow:::hipercow_root(root)$path$root
   path_script_abs <- file.path(path_root, path_script)
   dir.create(dirname(path_script_abs), FALSE, TRUE)
   bootstrap <- read_template("bootstrap.R")
+  prefix <- if (platform == "windows") "I:" else "/wpia-hn/Hipercow"
+  suffix <- if (platform == "windows") "" else "-linux"
+
+  bootstrap_repos <- c("https://mrc-ide.r-universe.dev",
+                       "https://cloud.r-project.org")
+  bootstrap_repos <- sprintf('c("%s")',
+                             paste0(bootstrap_repos, collapse = "\", \""))
+
+  bootstrap_pkgs <- c("hipercow", "remotes", "pkgdepends", "renv", "rrq")
+  deps <- pkgdepends::pkg_deps$new(bootstrap_pkgs)
+  deps$resolve()
+  bootstrap_pkgs <- deps$get_resolution()$package
```

Comment:
can you pull this bit into a small function

Reply:
Done